### ISSUE TITLE
language support for SE, JP, NO, NL and FI

### ISSUE THREAD

**Issue by kpotschi** [flybywiresim/a32nx#5357]
### Aircraft Version

Development

### Description

The current version doesn't come with lokPak files for the newly implemented MSFS languages.

### References (optional)

checked experimental branch and current open issues.

### Discord Username (optional)

MelodyConnor

**Comment by kpotschi** [comment]
Is there a common way to approach these kind of issues?

I thought about going for a Google Doc (one for each language) and find people from the community to translate, then have a second person review it for each language. I could then make a PR with the new files.

**Comment by kpotschi** [comment]
NL ✅

**Comment by davidwalschots** [comment]
> Is there a common way to approach these kind of issues?

Not really. But I think your idea on the matter is solid. So I'd go for that!

**Comment by kpotschi** [comment]
NO ✅

**Comment by kpotschi** [comment]
FI ✅

**Comment by loffa** [comment]
SE ✅

**Comment by kpotschi** [comment]
> SE ✅

hey there. 
where can i find it?

**Comment by kpotschi** [comment]
almost done with all the translations, i’ll create the PR this weekend and make sure it’s all nice and tidy. still looking for a reviewer for the japanese version.

**Comment by loffa** [comment]
> > SE ✅
> 
> hey there.
> where can i find it?

Sorry, I was mistaken. I just wanted to say that I can do the SE translations. Just need to find where to input them first.
I thought that there was a Google Doc link or similar that will be posted since your comments about that.
Please write in Discord  loffa#4108

**Comment by kpotschi** [comment]
> > > SE ✅
> > 
> > 
> > hey there.
> > where can i find it?
> 
> Sorry, I was mistaken. I just wanted to say that I can do the SE translations. Just need to find where to input them first.
> I thought that there was a Google Doc link or similar that will be posted since your comments about that.
> Please write in Discord loffa#4108

no worries! swedish is almost done, but i’ll reach out to you if necessary :)

**Comment by kpotschi** [comment]
JP ✅

**Comment by kpotschi** [comment]
SE ✅

I've got all translations ready now, thanks to a super cool community.

Created the files and I'm basically ready to test them, I ran into a couple of minor hickups:

- MSFS crashed about every third attempt at me loading a flight. I don't believe the locPaks to be breaking it, but I also haven't used MSFS in a couple weeks - probably an issue with the new Update #5?
- I'm not sure that all the tooltips will be needed, as there seems to be defaults set by MSFS. But this is something I gotta try properly and clean up the files if necessary
- I can't find the place where the _Aircraft Description_ is supposed to be displayed.

**Comment by bouveng** [comment]
> Created the files and I'm basically ready to test them, I ran into a couple of minor hickups:

@kpotschi Create a DRAFT pull-request of your work. And check in your files from your branch. Most of the conversation above should ideally be there instead. Then it becomes easy to se what you have done, see what is missing and suggest code changes.

The issue-ticket is more a discussion about the bug or feature, and not so much about technical details.

**Comment by kpotschi** [comment]
> > Created the files and I'm basically ready to test them, I ran into a couple of minor hickups:
> 
> @kpotschi Create a DRAFT pull-request of your work. And check in your files from your branch. Most of the conversation above should ideally be there instead. Then it becomes easy to se what you have done, see what is missing and suggest code changes.
> 
> The issue-ticket is more a discussion about the bug or feature, and not so much about technical details.

will do. thanks.
